src/@next/Banner/BannerStyle.ts

```diff
@@ -72,20 +72,86 @@ export const StyledBanner = styled.div`
     fill: ${Red.B93};
   }
 
+  &[data-type='fixed'],
+  &[data-type='fixed-noicon'] {
+    position: fixed;
+    border: 0;
+    border-radius: 0;
+    left: 0;
+    right: 0;
+    height: 56px;
```

Comment:
I see the banner measuring 100px in height. Seemingly, it is being overridden by the min-height attribute which is set to 100px.
![image](https://github.com/glints-dev/glints-aries/assets/12844836/25828301-7d1c-494e-aa27-732c674ee881)